app/src/main/java/com/example/androidacademyhomework/UI/MovieListFragment.kt

```diff
@@ -20,13 +29,43 @@ class MovieListFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        recyclerMovieList = view.findViewById(R.id.recyclerMovieList)
+        initRecycler()
+    }
+
+    override fun onStart() {
+        super.onStart()
+        updateData()
```

Comment:
I'm not sure if you want to load data in adapter on every start. Please read about fragment lifecycle.